homeassistant/components/motionblinds_ble/entity.py

```diff
@@ -0,0 +1,53 @@
+"""Base entity for the Motionblinds BLE integration."""
+
+import logging
+
+from motionblindsble.const import MotionBlindType
+from motionblindsble.device import MotionDevice
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity import Entity, EntityDescription
+
+from .const import CONF_BLIND_TYPE, CONF_MAC_CODE, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class MotionblindsBLEEntity(Entity):
+    """Base class for Motionblinds BLE entities."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    device: MotionDevice
+    entry: ConfigEntry
+
+    def __init__(
+        self,
+        device: MotionDevice,
+        entry: ConfigEntry,
+        entity_description: EntityDescription,
+        unique_id_suffix: str | None = None,
+    ) -> None:
+        """Initialize the entity."""
+        self._attr_unique_id: str = (
+            entry.data[CONF_ADDRESS]
+            if unique_id_suffix is None
+            else f"{entry.data[CONF_ADDRESS]}_{unique_id_suffix}"
```

Comment:
Please unpack ternaries that span multiple lines into an if block (or condense them so they don't)